concent_api/core/views.py

```diff
@@ -112,10 +118,18 @@ def receive(request, _message):
             current_time <= decoded_message_data.task_to_compute.compute_task_def['deadline'] + 2 * settings.CONCENT_MESSAGING_TIME and
             client_public_key == last_undelivered_message_status.message.auth.provider_public_key_bytes
         ):
+            set_message_as_delivered(last_undelivered_message_status)
```

Comment:
@Jakub89 This should be moved to branch with "return 204".

Reply:
I think it should stay, without it We can return that AckReportComputedTask message as many times as we want, as long as timing is right. It is a fix unrelated to usecase, but still needed.